### ISSUE TITLE
Disable autoplay option for media player

### ISSUE THREAD

**Issue by wondermart** [The-Devoted/devoted-wp-start#38]
## Objective

Disable the autoplay option for the media/video player block, to ensure no editors can opt to autoplay video (an accessibility requirement)

### Update

I originally thought we could set this control at the theme level, but it doesn't appear to be something that can be controlled globally. It can only be set as a boolean on each individual block instance (when included in a pattern or dropped in from the block directory). As a result, the outcome from this issue should be to put notes in docs & training so site editors know not to enable autoplay on videos.

- [x] In dev checklists
- [x] In wiki
